src/style.css

```diff
@@ -0,0 +1,72 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.list-nav {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  height: 100%;
+
+  list-style: none;
+}
+
+.list-nav > :first-child {
+  margin-left: 0;
+}
+
+.list-item {
+  display: flex;
+  margin-left: 20px;
+}
+
+.link-item {
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+  line-height: 60px;
+  color: #000;
+
+  position: relative;
+
+  transition: all 0.2s;
+}
+
+.link-item:visited {
+  color: #00acdc;
+}
+
+.link-item:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
```

Comment:
You can group it in one selector. Don't repeat yourself

Reply:
> You can group it in one selector. Don't repeat yourself

Ok, always forget about it